src/node_buffer.cc

```diff
@@ -730,12 +730,20 @@ void ReadDoubleBE(const FunctionCallbackInfo<Value>& args) {
 
 
 template <typename T, enum Endianness endianness>
-uint32_t WriteFloatGeneric(const FunctionCallbackInfo<Value>& args) {
+void WriteFloatGeneric(const FunctionCallbackInfo<Value>& args) {
+  Environment* env = Environment::GetCurrent(args);
+
+  bool noAssert = args[3]->BooleanValue();
+  if (!noAssert) {
```

Comment:
It is convention to use snake_case in native code, and since I've been bitten in the butt by it before (in buffer code actually) I'll have to agree with @thefourtheye's observation of avoiding double negative is a better option.

Also, we'll need to do some benchmarking on `BooleanValue()`. Last time I used it (way way back in v0.10) the performance was bad enough that we actually prefixed the check with `IsUndefined()`. So may look something like this:
```cpp
bool should_assert = !args[3]->IsUndefined() && !args[3]->BooleanValue();
```
Think that's right, but it's getting late.